std/digest/hmac.d

```diff
@@ -29,18 +31,19 @@ import std.meta : allSatisfy;
  * This type conforms to $(REF isDigest, std,digest,digest).
  */
 
-version(StdDdoc)
-/// Computes an HMAC over data read from stdin.
+/// Compute HMAC over an input string
 @safe unittest
 {
-    import std.digest.hmac, std.digest.sha, std.stdio;
+    import std.ascii : LetterCase;
+    import std.digest.digest : toHexString;
+    import std.digest.sha : SHA1;
```

Comment:
It's time we move on with https://github.com/dlang/phobos/pull/5013 and add a `package.d` to `std.digest` ...